weave/trace_server/interface/base_object_classes/human_annotation_column.py

```diff
@@ -0,0 +1,16 @@
+from typing import Optional
+
+from weave.trace_server.interface.base_object_classes import base_object_def
+
+
+class HumanAnnotationColumn(base_object_def.BaseObject):
+    json_schema: dict
```

Comment:
this is probably fine for now. you could add a field validator that uses https://python-jsonschema.readthedocs.io/en/latest/validate/ to check for a schema error